File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        map={}
        for i in range(len(nums)):
            d=target-nums[i]
            if d in map:
                return [map[d],i]
            map[nums[i]]=i
        return []

File: 0001-two-sum/0001-two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        var mp= new Dictionary<int,int>();
        for(int i=0;i<nums.Length;i++){
            int need=target-nums[i];
            if(mp.TryGetValue(need,out int j)) return new int[] {j,i};
            mp[nums[i]]=i;
        }
        return new int[] {};
    }
}